### DISCUSSION TITLE
我成功部署并且正常使用，但是几个月后我意外地不能进入网站

### DISCUSSION THREAD

**Discussion by Emotion04** [qkqpttgf/OneManager-php#719] [Q&A] [answered]
如题。网站页面全白，且某些浏览器会提示加载失败。开飞机或者使用vercel原域名也不能解决问题，vercel方面未报告异常

**Comment by qkqpttgf** [comment, marked as answer]
不在群吗……上周就有人有问题了。
vercel-php总会这样，请查看并复制你的config，然后重新覆盖部署。
